tests/unit/model-test.js

```diff
@@ -427,6 +427,107 @@ test("changedAttributes() works while the record is being updated", function(ass
   });
 });
 
+if (isEnabled('ds-reset-attribute')) {
+  test("resetAttribute() reverts a single attribute to its canonical value", function(assert) {
+    assert.expect(5);
+    env.adapter.shouldBackgroundReloadRecord = () => false;
+
+    run(function() {
+      store.push({
+        data: {
+          type: 'person',
+          id: '1',
+          attributes: {
+            name: 'Peter',
+            isDrugAddict: true
+          }
+        }
+      });
+
+      store.findRecord('person', 1).then(function(person) {
```

Comment:
Let's use `peekRecord` here: by this we can get rid of the `.then` and overwriting the `shouldBackgroundReloadRecord()`, which makes the test more readable.